### DISCUSSION TITLE
Almost no Sustain in Carla in VPO Strings SFZ's?

### DISCUSSION THREAD

**Discussion by jebofponderworthy** [falkTX/Carla#1457] [Q&A]
I just discovered Carla's new capabilities of SFZ via SFZero with considerable joy -- but then found that I could not sustain any of the VPO SFZ's for strings very long by holding keys down.  Is there a trick I'm missing?

**Comment by falkTX** [comment]
Carla SFZ support is very subpar, I recommend to use sfizz within carla instead.
At some point I might integrate that directly, but this is not done yet.
